stripe/res/values/dimens.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="card_icon_padding">12dp</dimen>
+    <dimen name="card_widget_min_width">320dp</dimen>
```

Comment:
We should have had a min width anyhow, but the calculations here will go bonkers if you make things too small.
320 is the smallest non-watch view I saw on the [Material devices guide](https://material.io/devices/). It would look pretty crowded, but could theoretically function at this value.

Though not if the user has the screen in giant text mode.